agreement_legal/__manifest__.py

```diff
@@ -18,6 +18,7 @@
         "data/module_category.xml",
         "data/agreement_stage.xml",
         "data/agreement_type.xml",
+        "demo/demo.xml",
```

Comment:
@ygol I meant something like this:
https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L38